js/renovation/ui/scroll_view/__tests__/scrollable.test.tsx

```diff
@@ -1075,44 +1084,150 @@ each([{
+                                    initStyles((viewModel).contentRef, contentSize);
+
+                                    setScrollbarPosition(viewModel.horizontalScrollbarRef,
+                                      scrollbarPosition);
+                                    setScrollbarPosition(viewModel.verticalScrollbarRef,
+                                      scrollbarPosition);
+
+                                    viewModel.cachedVariables.locked = locked;
+
+                                    let expectedValidationResult;
+                                    if (disabled || locked) {
+                                      expectedValidationResult = false;
+                                    } else if (bounceEnabled) {
+                                      expectedValidationResult = true;
+                                    } else if (isDxWheelEvent) {
+                                      expectedValidationResult = (contentSize > containerSize)
+                                        && (
+                                          (scrollbarPosition < 0 && delta > 0)
+                                          || (scrollbarPosition >= 0 && delta < 0)
+                                        );
+                                    } else if (!scrollByContent && !isScrollbarClicked) {
+                                      expectedValidationResult = false;
+                                    } else {
+                                      expectedValidationResult = containerSize < contentSize
+                                        || bounceEnabled;
+                                    }
+
+                                    const target = isScrollbarClicked
+                                      ? viewModel.containerRef.querySelector(`.${SCROLLABLE_SCROLLBAR_CLASS}`)
+                                      : viewModel.containerRef;
+                                    const e = { ...defaultEvent, target, delta };
+                                    if (isDxWheelEvent) {
+                                      (e as any).type = 'dxmousewheel';
+                                    }
+
+                                    expect(viewModel.cachedVariables.validateWheelTimer)
+                                      .toBe(undefined);
+
+                                    const actualResult = (viewModel).validate(e);
+                                    expect(actualResult).toBe(expectedValidationResult);
+
+                                    const isCheckedByTimeout = isDxWheelEvent
+                                      && expectedValidationResult && !bounceEnabled;
+
+                                    if (isCheckedByTimeout) {
+                                      expect(viewModel.cachedVariables.validateWheelTimer)
+                                        .not.toBe(undefined);
+                                    }
+
+                                    if (isCheckedByTimeout && (callCount === 2)) {
```

Comment:
Does we need 'callCount'?

Reply:
You are right, it is unnecessary, because I can check the expected result twice without this variable